.github/workflows/continuous-integration.yml

```
@@ -0,0 +1,71 @@
+name: Continuous Integration
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    strategy:
+      matrix:
+        node-version: [12, 14, 16, 17]
+      fail-fast: false
+    services:
+      postgres:
+        image: postgres:latest
+        env:
+          POSTGRES_HOST_AUTH_METHOD: trust
+        ports:
+          - 5432:5432
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install PostgreSQL Tools
+        run: sudo apt install postgresql-client
+      - name: Install dependencies
+        run: npm ci --ignore-scripts
+      - name: Load PostgreSQL schema
+        run: psql -h 127.0.0.1 -U postgres < ./test/schema.sql
+      - name: Run tests
+        run: npm test --ignore-scripts
```

As per-https://github.com/loopbackio/loopback-connector-postgresql/pull/499#discussion_r852447000, we can't reliably use `npm test --ignore-scripts` or `npm run <script name> --ignore-scripts` due to inconsistencies across NPM versions. We should create a `test:ci` script (which would not have any lifecycle scripts) and use that instead:

```suggestion
        run: npm run test:ci
```